src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
but this is explicitly documented as being unsupported in Mink: https://github.com/minkphp/Mink/blob/master/src/Driver/DriverInterface.php#L87